app/(authenticated)/leads/create/page.tsx

```diff
@@ -9,10 +9,10 @@ const LeadsCreatePage = () => {
   return (
     <div className='bg-white rounded-[10px] p-6'>
       <div className='flex items-center gap-5'>
-        <p onClick={() => router.back()} className=' cursor-pointer'>
+        <p onClick={() => router.push('/leads')} className=' cursor-pointer'>
```

Comment:
Instead of using `/leads`, use `PAGE_ROUTES` constant from `common-constant.ts` file.